CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
 cmake_minimum_required(VERSION 3.10)
 
+if (CMAKE_VERSION VERSION_GREATER 3.15)
```

Comment:
Well, the `CMP0091` policy exists since version 3.15. By having the number explicit it's clearer that when ccache is upgrading the `cmake_minimum_version` to something bigger than 3.15 the code can go away.

With a `if (POLICY CMP0091)` is not that clear, would require a comment.

Reply:
Ok, makes sense, in that case it should be:

```cmake
if(NOT CMAKE_VERSION VERSION_LESS 3.15)
  cmake_policy(SET CMP0091 NEW) # Needed by CMAKE_MSVC_RUNTIME_LIBRARY
endif()
```

, so that it works for exactly version `3.15`.